app/src/main/java/com/iron/espresso/ext/ViewExt.kt

```diff
@@ -20,4 +20,9 @@ fun View.invisibleIf(isInvisible: Boolean?) {
 @BindingAdapter("android:goneIf")
 fun View.goneIf(isGone: Boolean?) {
     this.isGone = isGone ?: false
+}
+
+@BindingAdapter("android:activationIf")
+fun View.activationIf(activation: Boolean?) {
+    this.alpha = if (activation == true) 1F else 0.4F
```

Comment:
```suggestion
    this.alpha = if (activation == true) 1f else 0.4f
```
문제는 아닌데 통일성을 위해 소문자로 작성하면 좋을 듯 합니다.

Reply:
수정하였습니다